executor/table_reader.go

```diff
@@ -96,6 +96,9 @@ func (e *TableReaderExecutor) Open(ctx context.Context) error {
 	}
 
 	e.resultHandler = &tableResultHandler{}
+	if e.feedback != nil && e.feedback.Hist() != nil {
+		e.ranges = e.feedback.Hist().SplitRange(e.ranges)
+	}
```

Comment:
Got it, please add the comment here to inform that `SplitRange` must be placed before `splitRanges` and explain the reasons.